test/client/submit.js

```diff
@@ -657,19 +650,12 @@ describe('client submit', function() {
           if (err) return done(err);
           doc2.create({age: 5}, function(err) {
             if (err) return done(err);
-            doc.pause();
-            var calledBack = false;
-            doc.on('error', function() {
```

Comment:
Is this emission actually expected, given the callback is called? (See above comment)